src/api/Shrooms.Contracts/Infrastructure/ISortableProperties.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Shrooms.Contracts.Infrastructure
+{
+    public interface ISortableProperties
```

Comment:
Can it be named just `ISortable` (like existing `IPageable`)?
Currently it's implemented by DTO/ViewModels, which have a lot of properties - so interface name is confusing.

Reply:
Renamed.